client/src/modules/chapters/pages/chaptersPage.tsx

```diff
@@ -1,8 +1,8 @@
 import { Heading, VStack, Stack } from '@chakra-ui/layout';
 import { NextPage } from 'next';
 import React from 'react';
+import { Grid, GridItem } from '@chakra-ui/react';
 
-import { ChapterCard } from 'components/ChapterCard';
```

Comment:
```
import { ChapterCard } from 'components/ChapterCard';
```

Reply:
I kept getting eslint error after adding the import statement back. I'll get back to work on this on Monday.

Reply:
![Screen Shot 2022-08-22 at 12 26 51 PM](https://user-images.githubusercontent.com/52594844/186002957-f788aecd-b4a4-4a97-ba51-482e7bf4ed70.png)
![Screen Shot 2022-08-22 at 12 27 39 PM](https://user-images.githubusercontent.com/52594844/186002960-6bf11462-3894-4e4a-b5f7-ed56bd743133.png)

@Sboonny 
I kept getting the same issue even though i added the key prop to the outer most element, is there anything wrong with eslint version? I did some research and found them here. https://github.com/jsx-eslint/eslint-plugin-react/issues/3215

Reply:
Map is a higher function that creates an array, that react requires key for it.

Placing key in header should please eslint, if it still isn't accepting the changes, I will open the pull request in the morning, if you don't mind This.

Reply:
@Sboonny I think I got it to work now with the new commit. Please have a look and let me know if you need anything to be done from my end.